frontend/src/shared/api.js

```diff
@@ -36,6 +36,8 @@ const API = {
   partners: createEndpoint('admin/partners/$id/'),
   products: createHyperlinkedEndpoint('admin/products/$id/'),
   productImages: createEndpoint('admin/products/$id/images/'),
+  users: { get: (data) => axiosInstance.get('admin/users/', data)},
```

Comment:
this API isn't usable without being an admin I'll update your branch so that api-token-auth returns a user object